### DISCUSSION TITLE
setup keycloak with reverse proxy ngnix

### DISCUSSION THREAD

**Discussion by flos777** [outline/outline#5026] [Self Hosting]
I try to set up outline with oidc keycloak ,
all my infrastructure is on docker, for the initial setup I used the tutorial proposed by : 
[https://blog.gurucomputing.com.au/doing-more-with-docker/deploying-outline-wiki/]

here are my different conf files :

docker-compose.yml :
- OIDC_CLIENT_SECRET=${AUTH_CLIENT_SECRET}
- OIDC_AUTH_URI=https://auth.mydomain/auth/realms/master/protocol/openid-connect/auth
- OIDC_TOKEN_URI=https://auth.mydomain/auth/realms/master/protocol/openid-connect/token
- OIDC_USERINFO_URI=https://auth.mydomain/auth/realms/master/protocol/openid-connect/userinfo
- OIDC_USERNAME_CLAIM=email
- OIDC_DISPLAY_NAME=Keycloak
- OIDC_SCOPES=openid profile email

Keycloak client :
![image](https://user-images.githubusercontent.com/59888599/224685427-f86b55eb-60be-443b-a458-e06b1728f28b.png)

ngnix config :
![outline mydomain](https://user-images.githubusercontent.com/59888599/224684664-2fa54abd-f45a-48d1-81b7-ca58217b42ab.png)

the problem is when I try to connect I get the following page :

![image](https://user-images.githubusercontent.com/59888599/224684946-bdad9054-5611-4314-afa7-263d0f8381b8.png)

![image](https://user-images.githubusercontent.com/59888599/224685225-14708f3d-9d4f-4156-8abf-38ea926f4a72.png)

**Comment by k1ck3r** [comment]
first of all, that's not an nginx.conf, it's a screenshot
second, you obviously need to re-read Keycloak's docs

whatever oid provider i used with outline it works flawlessly - you just need to adjust Keycloak's URLs and configs
i won't provide information where you are mistaken, because providing it won't point you out of your missing knowledge.

don't expect with copy/paste from guides everything to work out.

ps. you are posting issue obviously with Keycloak in Outline repo. that's my 2 cents on your issue. good luck
